<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QWidget" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="borrowPushButton">
         <property name="text">
          <string>借书</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="returnPushButton">
         <property name="text">
          <string>还书</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>图书编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>图书名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>图书作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>图书类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>借阅日期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>应还日期</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
